server/src/pages/shot/text-tool.js

```diff
@@ -119,6 +132,11 @@ exports.TextTool = class TextTool extends React.Component {
     prevDragMousePos = this.captureMousePosition(e);
   }
 
+  onDrop(e) {
+    // Block drop event inside text input
+    e.preventDefault();
+  }
```

Comment:
I don't think we need this now that it's a text `input`.

Reply:
We need this for stopping drop of formatted text from a text editor as shown in #4662. Apparently text input allows it

Reply:
But the dropped content is not formatted and doesn't break anything.  It's nice being able to drag some text into the input.

Reply:
it sure work now when I tried :)